src/VivViewer.js

```diff
@@ -4,6 +4,17 @@ import { OrthographicView } from '@deck.gl/core';
 import { VivViewerLayer, StaticImageLayer } from './layers';
 import VivViewerOverview from './VivViewerOverview';
 
+// Taken from https://stackoverflow.com/a/31732310/8060591
+function checkIsSafari() {
+  return (
+    navigator.vendor &&
+    navigator.vendor.indexOf('Apple') > -1 &&
+    navigator.userAgent &&
+    navigator.userAgent.indexOf('CriOS') === -1 &&
+    navigator.userAgent.indexOf('FxiOS') === -1
+  );
+}
```

Comment:
This would be a change from the original plan, but do you know what capability Safari is missing that causes the problem? And if so, could we check that, instead of trying to browser sniff? That would be a little more robust.